transformer_engine/jax/cpp_extensions.py

```diff
@@ -2162,8 +2183,18 @@ def lowering(ctx, qkv, softmax_aux, doutput, cu_seqlen, *, attn_bias_type, attn_
 
         args = CustomCallArgsWrapper(out_types, operands, operand_shapes)
         opaque = transformer_engine_jax.pack_fused_attn_descriptor(
-            batch, num_head, max_seqlen, max_seqlen, head_dim, scaling_factor, dropout_probability,
-            attn_bias_type, attn_mask_type, jax_dtype_to_te_dtype(qkv_aval.dtype), is_training)
+            batch,
+            num_head,
+            max_seqlen,
+            max_seqlen,
+            head_dim,
+            0,    # there is no rng_state in backward currently
```

Comment:
The backward need to know which element was droped.
How this is done? Maybe extend the comment to specify this?

Reply:
The dropped elements in the softmax_aux tensor are encoded as negative values. This encoding trick is used because the elements after the softmax operation are always greater than 0. By marking the dropped elements as negative values, we can differentiate them from the non-dropped elements. I will enhance the comment here.

Reply:
Comment is enhanced in c8615cf3866fa0266d20739ef221b1ccdb5e45ea